storage_file/models/ir_actions_report.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Camptocamp SA (http://www.camptocamp.com).
+# @author Simone Orsi <simahawk@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class IrActionsReport(models.Model):
+    _inherit = "ir.actions.report"
+
+    def render_qweb_pdf(self, res_ids=None, data=None):
+        return super(
+            IrActionsReport, self.with_context(print_report_pdf=True)
+        ).render_qweb_pdf(res_ids=res_ids, data=data)
```

Comment:
```suggestion
        self = self.with_context(print_report_pdf=True)
        return super().render_qweb_pdf(res_ids=res_ids, data=data)
```
Not sure what you think but I recently discovered this is the same and it's more readable.
Not blocking of course